config/services.yml

```diff
@@ -845,11 +673,9 @@ https://cmr.uat.earthdata.nasa.gov:
       name: http
       params:
         url: http://localhost:4000/example/harmony
-    collections:
-      # AMSR-E/Aqua L2B Global Swath Ocean Products derived from Wentz Algorithm V086
-      # Good example, being in UAT, global, many public non-global granules,
-      # and not yet on the services roadmap.
-      - id: C1104-PVC_TS2
+    umm_s:
+      - S1233603903-EEDTEST
```

Comment:
We might want to update this description for the UMM-S record to indicate it's for testing harmony http service integration (the service isn't docker-based or make GDAL calls - it does not actually do anything).

> Access to Harmony internal Docker-based GDAL service for AMSR-E/Aqua L2B Global Swath Ocean Products.